### PR TITLE
Fixing bugs in arastorage and augumenting testcasse to increase code coverage.

### PR DESCRIPTION
**Pull request by shivgarg**

### CODE REVIEW

File: apps/examples/testcase/ta_tc/arastorage/utc/utc_arastorage_main.c [hunk 1]
```diff
@@ -35,285 +36,263 @@
...
-	res = db_exec(query);
-	if (DB_ERROR(res)) {
-		printf("db_exec Failed(Create Attribute) Name : %s Type : DOUBLE\n",
-			   g_attribute_set[1]);
-		g_arastorage_tc_fail_count++;
-		return;
-	}
 
-	for (i = 0; i < DATA_SET_NUM; i++) {
+	for (; i < DATA_SET_NUM*DATA_SET_MULTIPLIER; i++) {
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/arastorage/utc/utc_arastorage_main.c]: M*D -> M * D

File: apps/examples/testcase/ta_tc/arastorage/utc/utc_arastorage_main.c [hunk 2]
```diff
@@ -35,285 +36,263 @@
...
-
-const static char *g_attribute_set[] = {"id", "date", "fruit", "weight"};
+const static char *g_attribute_set[] = {"id", "date", "fruit", "value", "weight"};
 
 struct arastorage_data_type_s {
-	int int_value;
 	long long_value;
 	char *string_value;
 	double double_value;
+
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/arastorage/utc/utc_arastorage_main.c]: unnecessary empty line

File: apps/examples/testcase/ta_tc/arastorage/utc/utc_arastorage_main.c [hunk 3]
```diff
@@ -35,285 +36,263 @@
...
-				   g_arastorage_data_set[i].string_value, g_arastorage_data_set[i].double_value,
-				   i);
-			g_arastorage_tc_fail_count++;
-			return;
-		}
+		TC_ASSERT("db_exec", DB_SUCCESS(res));
 	}
 #else
-	for (i = 0; i < DATA_SET_NUM; i++) {
+	for (; i < DATA_SET_NUM*DATA_SET_MULTIPLIER; i++) {
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/arastorage/utc/utc_arastorage_main.c]: same as previous

File: apps/examples/testcase/ta_tc/arastorage/utc/utc_arastorage_main.c [hunk 4]
```diff
@@ -35,285 +36,263 @@
...
 		memset(query, 0, QUERY_LENGTH);
-		snprintf(query, QUERY_LENGTH, "INSERT (%d, %ld, \'%s\') INTO %s;",
-				 g_arastorage_data_set[i].int_value, g_arastorage_data_set[i].long_value,
-				 g_arastorage_data_set[i].string_value, RELATION_NAME);
+		snprintf(query, QUERY_LENGTH, "INSERT (%d, %ld, \'%s\', %d) INTO %s;",
+				 i, g_arastorage_data_set[i%DATA_SET_NUM].long_value,
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/arastorage/utc/utc_arastorage_main.c]: i%D -> i % D

File: apps/examples/testcase/ta_tc/arastorage/utc/utc_arastorage_main.c [hunk 5]
```diff
@@ -35,285 +36,263 @@
...
+				 g_arastorage_data_set[i%DATA_SET_NUM].string_value, 1000 - i, RELATION_NAME);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/arastorage/utc/utc_arastorage_main.c]: same

File: framework/src/arastorage/index_manager.c [hunk 6]
```diff
@@ -478,14 +480,14 @@ db_result_t db_indexing(relation_t *rel)
 
 	for (tuple_id = 0; tuple_id < cardinality; tuple_id++) {
 		memset(row, 0, sizeof(row));
+		DB_LOG_V("DB: Indexing Tuple id %d\n",tuple_id);
```
Comment (sunghan-chang) [inline framework/src/arastorage/index_manager.c]: space after ,

File: apps/examples/testcase/ta_tc/arastorage/utc/utc_arastorage_main.c [hunk 7]
```diff
@@ -35,285 +36,262 @@
...
-	{ 2,        20160102,       "banana",          2.0002},
-	{ 3,        20160103,       "mango",           3.0003},
-	{ 4,        20160104,       "orange",          4.0004},
-	{ 5,        20160105,       "melon",           5.0005},
-	{ 6,        20160106,       "kiwi",            6.0006},
-	{ 7,        20160107,       "peach",           7.0007},
-	{ 8,        20160108,       "cherry",          8.0008},
-	{ 9,        20160109,       "strawberry",      9.0009},
-	{10,        20160110,       "watermelon",     10.0010}
+const static struct arastorage_data_type_s g_arastorage_data_set[DATA_SET_NUM] = {
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/arastorage/utc/utc_arastorage_main.c]: You make type definition, arastorage_data_t, but not used. Let's remove arastorage_data_t or change this to _t.

**Comment by sunghan-chang** [comment]
@shivgarg I don't have any conversation anymore. You can get a review about operation from Taejun.

**Comment by shivgarg** [comment]
@Taejun-Kwon Please review this pull request.

**Pull request merged**